libs/application/templates/party-application/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+import { Constituencies } from './types'
+
+export const constituencyMapper: Record<
+  Constituencies,
+  {
+    low: number
+    high: number
+  }
+> = {
+  Norðausturkjördæmi: { low: 300, high: 400 },
```

Comment:
would probably be nice to leave a comment here on where and how you got those "magic" numbers, just for future references